src/external/os/Earthfile

```diff
@@ -62,3 +62,7 @@ ADDUSER:
     USER ${user}
     ENV HOME=/home/${user}
     WORKDIR ${workdir}
+
+FIXUSER:
+    FUNCTION
+    RUN rosdep fix-permissions
```

Comment:
@marcoshuck meta: considering this is ROS specific, I wonder if it wouldn't better be among `ros` specific Earthfiles.

Reply:
I ended up moving it to a new `rosdep` folder as `rosdep` is a standalone tool. Happy to move it somewhere else if you don't agree with having a specific folder.